components/list/list-item-mixin.js

```diff
@@ -451,6 +462,17 @@ export const ListItemMixin = superclass => class extends composeMixins(
 				margin-bottom: -12.5px;
 				margin-top: -11.5px;
 			}
+			:host([_list-item-new-styles][draggable][_hovering]) [slot="add"],
+			:host([_list-item-new-styles][draggable][_focusing]) [slot="add"],
+			:host([_list-item-new-styles][draggable][_hovering]) [slot="add-top"],
+			:host([_list-item-new-styles][draggable][_focusing]) [slot="add-top"],
```

Comment:
I switched this padding to apply whenever there is hover/focus and draggable and not just when selectable (since the border is shown whenever there is hover/focus now and not just on selectable)